sdk/typescript/src/cryptography/secp256k1-keypair.ts

```diff
@@ -1,17 +1,24 @@
 // Copyright (c) 2022, Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-import * as secp from "@noble/secp256k1";
+import * as secp from '@noble/secp256k1';
 import { Base64DataBuffer } from '../serialization/base64';
 import { Keypair } from './keypair';
 import { PublicKey, SignatureScheme } from './publickey';
 import { hmac } from '@noble/hashes/hmac';
-import { sha256 } from "@noble/hashes/sha256";
-import { Secp256k1PublicKey } from "./secp256k1-publickey";
+import { sha256 } from '@noble/hashes/sha256';
+import { Secp256k1PublicKey } from './secp256k1-publickey';
+import { HDKey } from '@scure/bip32';
+import bip39 from 'bip39-light';
+import {
+  isValidBIP32Path,
+  normalizeMnemonics,
+  validateMnemonics,
+} from './mnemonics';
 
 secp.utils.hmacSha256Sync = (key: Uint8Array, ...msgs: Uint8Array[]) => {
   const h = hmac.create(sha256, key);
-  msgs.forEach(msg => h.update(msg));
+  msgs.forEach((msg) => h.update(msg));
```

Comment:
do we need this parenthesis?

Reply:
removed all lint changes to a new PR. sdk/typescript needs to completely linted